alignak_module_backend/broker/module.py

```diff
@@ -137,12 +136,17 @@ def getToken(self, username, password, generatetoken):
             logger.exception("Exception: %s", exp)
             self.backend_connected = False
 
-    def backendConnection(self):
-        """
-        Backend connection to check live state update is allowed
+    def backendConnection(self, default_realm='All'):
+        """Backend connection to check live state update is allowed
 
         :return: True/False
         """
+        if not self.default_realm:
+            params = {'where': '{"name":"%s"}' % default_realm}
+            realms = self.backend.get('realm', params=params)
+            for realm in realms['_items']:
+                self.default_realm = realm['_id']
```

Comment:
Sure. The default realm is used to store the alignak program status and we only need one realm to post/patch the data. Nothing to deal with sub realms for this ...